lnst/RecipeCommon/Perf/Measurements/BaseFlowMeasurement.py

```diff
@@ -3,9 +3,8 @@
 from lnst.Controller.RecipeResults import ResultType
 from lnst.RecipeCommon.Perf.Measurements.MeasurementError import MeasurementError
 from lnst.RecipeCommon.Perf.Measurements.BaseMeasurement import BaseMeasurement
-from lnst.RecipeCommon.Perf.Measurements.BaseMeasurement import BaseMeasurementResults
-from lnst.RecipeCommon.Perf.Results import SequentialPerfResult
 from lnst.RecipeCommon.Perf.Results import ParallelPerfResult
+from lnst.RecipeCommon.Perf.Measurements.Results import FlowMeasurementResults, AggregatedFlowMeasurementResults
```

Comment:
Please swap these two lines, so that imports are grouped by modules.

Reply:
done